esmvalcore/preprocessor/_volume.py

```diff
@@ -275,6 +277,51 @@ def volume_statistics(cube, operator):
     return _create_cube_time(src_cube, result, times)
 
 
+def axis_statistics(cube, axis, operator):
+    """Perform statistics along a given axis.
+
+    Operates over an axis direction. If weights are required,
+    they are computed using the coordinate bounds.
+
+    Arguments
+    ---------
+    cube: iris.cube.Cube
+        input cube.
+    axis: str
+        direction over where to apply the operator.
+    operator: str
+        statistics to perform.
```

Comment:
Here, too, we need a list of supported operators.

Reply:
added as well, please let me know if the changes work for you